client/app/scripts/components/node-details/__tests__/node-details-table-test.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import TestUtils from 'react/lib/ReactTestUtils';
+import { Provider } from 'react-redux';
+import configureStore from '../../../stores/configureStore';
+
+// need ES5 require to keep automocking off
+const NodeDetailsTable = require('../node-details-table.js').default;
+
+describe('NodeDetailsTable', () => {
+  let nodes, columns, component;
+
+  beforeEach(() => {
+    columns = [
+      { id: 'kubernetes_ip', label: 'IP' },
+      { id: 'kubernetes_namespace', label: 'Namespace' },
+    ];
+    nodes = [
+      {
+        id: 'node-1',
+        metadata: [
+          { id: 'kubernetes_ip', label: 'IP', value: '10.244.253.24' },
```

Comment:
Good point, forgot to run the tests once more in the end! :) I modified `line 14`, so the tests should pass now (they weren't passing before).